includes/Modules/Idea_Hub.php

```diff
@@ -500,6 +500,18 @@ protected function setup_assets() {
 					),
 				)
 			),
+			new Script(
+				'googlesitekit-idea-hub-notice',
+				array(
+					'src'           => $base_url . 'js/googlesitekit-idea-hub-notice.js',
+					'dependencies'  => array(
+						'googlesitekit-vendor',
+						'googlesitekit-api',
+						'googlesitekit-data',
+					),
```

Comment:
Seems like we don't need these dependencies for this script, do we? Let's remove them then.